.gitignore

```diff
@@ -457,4 +457,5 @@ $RECYCLE.BIN/
 logs/
 
 #.env files should never be commited
-.env
\ No newline at end of file
+.env
+/src/OnlineSales/appsettings.json

```

Comment:
hi @WASandaruwan, why have you added appsettings.json file to .gitignore?

Reply:
Hi @peterliapin  - I needed to change the password for the posgress db, since the changes done to this file are shares with everyone I ignored it.